src/engine/SignConstraint.h

```diff
@@ -150,10 +150,41 @@ class SignConstraint : public PiecewiseLinearConstraint
     unsigned getB() const;
     unsigned getF() const;
 
+  bool supportPolarity() const;
+
+  /*
+    Return the polarity of the Sign Constraint, which computes how symmetric
+    the bound of the input to this ReLU is with respect to 0.
+    Let LB be the lowerbound, and UB be the upperbound.
+    If LB >= 0, polarity is 1.
+    If UB <= 0, polarity is -1.
```

Comment:
@anwu1219 the Sign in Marabou goes by the standard that sing(x)=1 iff x>=0 so I think it should be
    **If UB < 0, polarity is -1.**